lib/db/schema/patch-011-012.sql

```diff
@@ -0,0 +1,32 @@
+-- drop old stored procedures
+
+DROP PROCEDURE `createAccount_1`;
+
+DROP PROCEDURE `account_1`;
+
+DROP PROCEDURE `emailRecord_1`;
+
+DROP PROCEDURE `verifyEmail_1`;
+
+DROP PROCEDURE `forgotPasswordVerified_1`;
+DROP PROCEDURE `forgotPasswordVerified_2`;
+DROP PROCEDURE `forgotPasswordVerified_3`;
+DROP PROCEDURE `createAccountResetToken_1`;
```

Comment:
tiny nit: separate this from `forgotPasswordVerified_*` with a newline for consistency